aura/lib/Aura/Build.hs

```diff
@@ -157,7 +157,7 @@ cloneRepo pkg usr = do
   logDebug $ "Currently in: " <> displayShow currDir
   scriptsDir <- liftIO $ chown usr currDir [] *> clone pkg
   case scriptsDir of
-    Nothing -> pure . Left . Failure . buildFail_7 $ bName pkg
+    Nothing -> pure . Left . Failure . FailMsg.  buildFail_7 $ bName pkg
```

Comment:
whitespace mis-click?

Reply:
Looks like it, although luckily that doesn't affect compilation.